src/pages/FeaturePreviews.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash/fp'
+import { Fragment, useCallback, useEffect, useState } from 'react'
+import { div, h, h2, p, span } from 'react-hyperscript-helpers'
+import { Checkbox, PageBox, spinnerOverlay } from 'src/components/common'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { SimpleFlexTable } from 'src/components/table'
+import TopBar from 'src/components/TopBar'
+import { isFeaturePreviewEnabled, toggleFeaturePreview, useAvailableFeaturePreviews } from 'src/libs/feature-previews'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+
+
+export const FeaturePreviews = () => {
+  const { featurePreviews, loading, error } = useAvailableFeaturePreviews()
+
+  // The source of truth for whether or not a feature preview is enabled is `isFeaturePreviewEnabled`, which reads
+  // from local preferences. However, changing a local preference won't re-render this component and its checkboxes.
+  // Thus, we also need to store this in component state and keep in sync with local preferences.
+  const getFeaturePreviewState = useCallback(() => {
+    return _.flow(_.map(({ id }) => [id, isFeaturePreviewEnabled(id)]), _.fromPairs)(featurePreviews)
+  }, [featurePreviews])
```

Comment:
[Nit] It took my eyes some time to parse the function above, which motivated me to mention the following as an alternative:
```javascript
  const getFeaturePreviewState = useCallback(() => 
    _.flow(
      _.map(({ id }) => [id, isFeaturePreviewEnabled(id)]),
      _.fromPairs
    )(featurePreviews), [featurePreviews])
```

Reply:
Changed in https://github.com/DataBiosphere/terra-ui/pull/3382/commits/fac15158815516237b40681e4067c587b86cd49c.